<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/photo/logo.png</normaloff>:/photo/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>方正粗金陵简体</family>
           <pointsize>17</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>方正粗金陵简体</family>
           <pointsize>17</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="login_button">
         <property name="font">
          <font>
           <family>方正粗金陵简体</family>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>登  录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="register_button">
         <property name="font">
          <font>
           <family>方正粗金陵简体</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>注  册</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-20</y>
      <width>876</width>
      <height>632</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/photo/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
  <action name="actionquit">
   <property name="icon">
    <iconset theme="battery">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出计费系统</string>
   </property>
  </action>
  <action name="actionaddcard">
   <property name="text">
    <string>注册</string>
   </property>
   <property name="toolTip">
    <string>注册一个新的账号</string>
   </property>
  </action>
  <action name="actionlogin">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录您的账号</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="text">
    <string>注销</string>
   </property>
   <property name="toolTip">
    <string>注销账号</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
